src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -85,67 +86,85 @@ const ResearchProfile = () => {
   ) : !person ? (
     <NotFound />
   ) : (
-    <BackgroundDiv>
-      <Helmet>
-        <title>{fullName}</title>
-      </Helmet>
-      <Typography variant="h1" sx={{ mt: '1rem', mb: '2rem' }}>
-        {fullName}
-      </Typography>
-      <Typography variant="h2">{t('common.employments')}</Typography>
-      {activeAffiliations.length > 0 ? (
-        <Box sx={lineSx}>
-          <WorkIcon />
-          <Box sx={textContainerSx}>
-            {activeAffiliations.map(({ organization }) => (
-              <AffiliationHierarchy key={organization} unitUri={organization} commaSeparated />
-            ))}
+    <Box>
+      <Box sx={{ bgcolor: 'person.main', py: '1.1rem', display: 'flex', alignItems: 'center', gap: '1rem' }}>
+        <Typography variant="h1" sx={{ ml: '2rem' }}>
+          {fullName}
+        </Typography>
+        {orcidUri && <img src={orcidIcon} height="20" alt="orcid" />}
+      </Box>
+      <BackgroundDiv>
+        <Helmet>
+          <title>{fullName}</title>
+        </Helmet>
+
+        {activeAffiliations.length > 0 ? (
+          <Box sx={lineSx}>
+            <Box sx={{ display: 'flex', gap: '0.5rem' }}>
+              {activeAffiliations.map(({ organization, role }) => (
+                <Box
+                  sx={{
+                    border: '2px solid',
+                    borderColor: 'secondary.light',
+                    borderRightColor: 'primary.main',
+                    borderRightWidth: '1px',
+                    width: 'fit-content',
```

Comment:
Er det en grunn til å sette border rundt alt her? Hvis vi endrer bakgrunnsfargene senere må vi da huske å endre fargen på border her også
```suggestion
                    borderRight: '1px solid',
                    borderRightColor: 'primary.main',
```